src/droplet.F

```diff
@@ -449,22 +457,29 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
                           neighbor,num_fallout,errcode)
 
 #ifdef MPI
-      pdata_neighbor(np) = neighbor 
       if(neighbor .eq. mynorth) then 
+         pdata_neighbor(np) = inorth
```

Comment:
why we need this change instead of just using `pdata_neighbor(np) = neighbor`?

Reply:
@sjsprecious Good question.  I needed to redefine the meaning of pdata_neighbor because it allows us to identify particles that are on the border to be treated properly and to perform this logic in one place instead of multiple places.  It also eliminates the need to create individual mask logic variables so that we can call count_prefix.

Reply:
Thanks for the clarification.